apps/prairielearn/src/lib/courseUtil.js

```diff
@@ -14,7 +14,8 @@ function getCommitHash(coursePath, callback) {
     cwd: coursePath,
     env: process.env,
   };
-  exec('git rev-parse HEAD', execOptions, (err, stdout) => {
+  exec('git rev-parse HEAD', execOptions, (err, stdout, stderr) => {
+    console.log({ err, stdout, stderr });
```

Comment:
Was this unintentionally committed?

Reply:
Sorry, this ended up being added in a rush. That said, adding the `stderr` to the error log may be useful to help debugging future issues of similar problems. Does it make sense to add the `stderr` content as the data for an `error.makeWithData` in the error handling?

Reply:
Might be worth a shot! I'd recommend confirming that those properties are shown in the server job logs - I'm not sure off the top of my head if they will be or not.

Reply:
Done in e62c8bb40.

<img width="888" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/1004907/633ecca6-18a7-468b-adf1-7ccacad1ae95">